sections/security/limitrequests.md

```diff
@@ -16,7 +16,6 @@ app.enable('trust proxy');
 var apiLimiter = new RateLimit({
   windowMs: 15*60*1000, // 15 minutes
   max: 100,
-  delayMs: 0 // disabled

```

Comment:
[v3 changes](https://github.com/nfriedly/express-rate-limit#v3-changes) say that `delayAfter` and `delayMs` options were moved to a new module: [`express-slow-down`](https://npmjs.org/package/express-slow-down).

In addition to the current change, should we include something like the below as well?

```javascript
const slowDown = require('express-slow-down');
 
app.enable('trust proxy'); // only if you're behind a reverse proxy (Heroku, Bluemix, AWS if you use an ELB, custom Nginx setup, etc)
 
const speedLimiter = slowDown({
  windowMs: 15*60*1000, // 15 minutes
  delayAfter: 100 // allow 100 requests per 10 minutes, then...
  delayMs: 500 // begin adding 500ms of delay per request above 100:
  // request # 101 is delayed by  500ms
  // request # 102 is delayed by 1000ms
  // request # 103 is delayed bu 1500ms
  // etc.
});
 
//  apply to all requests
app.use(speedLimiter);
```

Reply:
I'm not sure it's a good practice to suggest slowing down request rates from the perspective of security, as well as that module not being very heavily used. IMO, I would lean more towards what we've got suggested in the PR currently

Reply:
Thanks for clarifying @js-kyle! I get what you are saying. 👍